app/src/App/index.js

```diff
@@ -90,6 +92,14 @@ const App = () => {
               />
             }
           />
+          <Route
+            path="/portfolios"
+            element={<Protected component={Portfolios} />}
+          />
+          <Route
+            path="/:portfolio_id"
```

Comment:
This is the only way I can make the error disappear for the time being. But I would like to know why I can't add anything before `/:portfolio_id`. 😄

Reply:
See #12.